.gitignore

```diff
@@ -45,3 +45,78 @@ $RECYCLE.BIN/
 Network Trash Folder
 Temporary Items
 .apdisk
```

Comment:
these are just things for git to ignore so that they don't get added to the repo